### DISCUSSION TITLE
Helm Chart for Kafka Cluster

### DISCUSSION THREAD

**Discussion by neuwerld3** [strimzi/strimzi-kafka-operator#4643] [General]
I'm new to Kubernetes and Helm and have a question.
How can I create a helm chart from the kafka-persistent.yaml file?

We are using ArgoCD which is a GitOPS tool that handles helm charts and I would like to deploy the Kafka Cluster using a helm chart in this tool.

Thanks in advance.

**Comment by scholzj** [comment]
Strimzi does not provide such Helm Chart. So you would need to write it your self. I think some people on Slack mentioned in the past writing similar Helm Charts. But I'm not sure whether they shared it somewhere or not.
